dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -212,6 +212,15 @@ class CRegFolderEnum :
         END_COM_MAP()
 };
 
+#define REGISTRYSHELLVIEWCOLUMNS 2
+
+enum registry_columns
+{
+    REGISTRY_COL_NAME,
+    REGISTRY_COL_TYPE,
+    REGISTRY_COL_VALUE,
+};
```

Comment:
It is common to expand an enum like this instead:
```suggestion
enum registry_columns
{
    REGISTRY_COL_NAME,
    REGISTRY_COL_TYPE,
    REGISTRY_COL_VALUE,
    REGISTRY_COL_MAX,
};
```

Then you'd have to check with `>=` instead of `>` of course.

Reply:
Pushed a commit, to address this.